docs/development/achievements.md

```diff
@@ -1,22 +1,12 @@
 # Achievements System
 
-## How To Enable
-Achievements are enabled and configured per theme.
-To enable the achievements system add `"achievements": true` to the game block of the theme config `themes/#####/config/config.json5`. e.g:
```

Comment:
This appears to be outdated- at least, it's not in the demo 'game' but it still has achievements. 

Looks like achievements are 'always on' and we can turn them off by passing an empty array in the achievements config.json5  ...updated here, but tell me if I'm wrong 😂

Reply:
Good spot! That changed recently when we scoped everything per theme. Feel free to edit.